llvm/lib/Transforms/IPO/HotColdSplitting.cpp

```diff
@@ -394,9 +382,16 @@ Function *HotColdSplitting::extractColdRegion(
   LLVM_DEBUG(dbgs() << "Split profitability: benefit = " << OutliningBenefit
                     << ", penalty = " << OutliningPenalty << "\n");
   if (!OutliningBenefit.isValid() || OutliningBenefit <= OutliningPenalty)
-    return nullptr;
+    return false;
+
+  return true;
+}
 
-  Function *OrigF = Region[0]->getParent();
+Function *HotColdSplitting::extractColdRegion(
```

Comment:
Please add comment to clarify how BB is sufficient to extract a region. Is it a single-entry single-exit region?